thehive/info.json

```diff
@@ -0,0 +1,2241 @@
+          "tooltip": "Select the input type for creating an observable. You can choose either 'Create Observable using Attachment' or 'Create Observable using Plain Text.'",
+          "visible": true,
+          "editable": true,
+          "required": true,
+          "options": [
+            "Autonomous-System",
+            "Domain",
+            "File",
+            "FQDN",
+            "Hash",
+            "IP",
+            "Mail",
+            "Mail_Subject",
+            "Other",
+            "Regexp",
+            "Registry",
+            "Uri_Path",
+            "URL",
+            "User-Agent"
+          ]
+        },
+        {
+          "title": "Message",
+          "name": "message",
+          "type": "text",
+          "description": "Specify the message of the observable you want to update for alert in TheHive Platform",
+          "tooltip": "Specify the message of the observable you want to update for alert in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Add Tags",
+          "name": "addTags",
+          "type": "text",
+          "description": "Specify CSV list of tags will be added to the current observable.",
+          "tooltip": "Specify CSV list of tags will be added to the current observable.",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Remove Tags",
+          "name": "removeTags",
+          "type": "text",
+          "description": "Specify CSV list tags will be removed from the current observable",
+          "tooltip": "Specify CSV list of tags will be removed from the current observable",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "TLP",
+          "name": "ignoreSimilarity",
+          "type": "checkbox",
+          "description": "Specify the TLP of the observable you want to update.",
+          "tooltip": "Specify the TLP of the observable you want to update.",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Additional Field",
+          "name": "additional_field",
+          "type": "json",
+          "description": "Specify the additional fields required for the use case. Provide the fields in JSON format.",
+          "tooltip": "Specify the additional fields required for the use case. Provide the fields in JSON format.",
+          "visible": true,
+          "editable": true,
+          "required": false
+        }
+      ],
+      "enabled": ""
+    },
+    {
+      "title": "Delete Observable",
+      "operation": "delete_observable",
+      "description": "Delete a specific observable from the TheHive platform based on the input parameters you have specified.",
```

Comment:
Please update the description: Delete a specific observable from the TheHive platform based on the observable ID you have specified.